src/structures/InviteStageInstance.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Base = require('./Base');
+const Collection = require('../util/Collection');
+
+class InviteStageInstance extends Base {
+  constructor(client, data, channelID, guildID) {
+    super(client);
+
+    /**
+     * The ID of the stage channel this invite is for
+     * @type {Snowflake}
+     */
+    this.channelID = channelID;
+
+    /**
+     * The guild ID of the stage channel
+     * @type {Snowflake}
+     */
+    this.guildID = guildID;
+
+    /**
+     * The members speaking in the stage channel
+     * @type {Collection<Snowflake, GuildMember>}
+     */
+    this.members = new Collection();
+
+    this._patch(data);
+  }
+
+  _patch(data) {
+    /**
+     * The topic of the stage instance
+     * @type {string}
+     */
+    this.topic = data.topic;
+
+    /**
+     * The number of users in the stage channel
+     * @type {number}
+     */
+    this.participantCount = data.participant_count;
+
+    /**
+     * The number of users speaking in the stage channel
+     * @type {number}
+     */
+    this.speakerCount = data.speaker_count;
+
+    for (const rawMember of data.members) {
```

Comment:
The members should be cleared when this is patched, to avoid stale data from persisting:
```suggestion
    this.members.clear();
    for (const rawMember of data.members) {
```